### DISCUSSION TITLE
No installer program

### DISCUSSION THREAD

**Discussion by Suggy50** [Open-Shell/Open-Shell-Menu#1849] [General]
To build Open-Shell, navigate to the Setup folder and run __MakeFinal.bat.
The final files (installers, archives) are saved to the Setup\Final folder.

I run the _MakeFinal.bat but theres no Final folder or installer to be found

**Comment by ge0rdi** [comment]
Most likely the build process ended with an error before `Final` folder was created.

Do you have `Wix` and `HTML Help Workshop` installed as they are needed for building the installer (see [BUILDME.txt](https://github.com/Open-Shell/Open-Shell-Menu/blob/master/Src/BUILDME.txt))?
